test/built-ins/Date/UTC/non-integer-values.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
```

Comment:
Who was this test written by?

Reply:
oh, sorry, forgot to change this line

Reply:
Could you make a follow-up PR to fix this line?

Reply:
@littledan , https://github.com/tc39/test262/pull/1393